packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -0,0 +1,53 @@
+import { makeZoeHelpers, defaultAcceptanceMsg } from './helpers/zoeHelpers';
+
+/*
+    Roles in the arrangement:
+    - Contract creator: describes the contract with:
+        - number of seats, show, date/time of start
+        - expected (ERTP) amount per ticket (we assume all tickets cost the same)
+    - Smart Contract: 
+        - mints the tickets
+        - provides the seats
+    - Auditorium (unique contract seat, usually taken by the contract creator): the person hosting 
+    the Opera show, selling the tickets and getting the payment back
+    - Ticket buyers (contract seat created on demand): 
+        - can see the available opera show seats
+        - can consult the terms
+        - can redeem the zoe invite with the proper payment to get the ticket back
+*/
+
+export const makeContract = harden(zoe => {
+
+  const {
+    rejectOffer,
+    checkIfProposal,
+    swap,
+    canTradeWith,
+    getActiveOffers,
+    assertKeywords,
+  } = makeZoeHelpers(zoe);
+
+
+  const auditoriumSeat = harden({
+    getSalesMoney(){
+      throw `TODO`
+    }
+  })
+  const auditoriumInvite = zoe.makeInvite(auditoriumSeat);
```

Comment:
```suggestion
  const { invite: auditoriumInvite } = zoe.makeInvite(auditoriumSeat);
```
I think you're returning a record of invite, inviteHandle to Zoe when you should just return the invite. I think that's why it is saying it's not a payment - because it's a record, not a payment

Reply:
oh what a trivial mistake :-/
i am grateful for the time you spent on this
i'll try to be more careful

Reply:
Happens to me all the time, and this exact thing might happen to a user, so it's all good. Thanks for working on this!